test_abnormality_detection.py

```diff
@@ -42,9 +47,13 @@ def parse_args() -> argparse.Namespace:
     image = Image.open(args.image_path).convert('RGB')
     # load allowed_regions map
     map_ = Image.open(args.allowed_regions).convert('L')
+    map_rgb = Image.open(args.allowed_regions)
     allowed_regions = np.asarray(map_)
     allowed_regions = allowed_regions.T
+    # load confidence threshold
+    conf_threshold = args.conf_threshold
 
+    # pass image and allowed_regions as arguments for judge_image()
     # judge image according to allowed_regions
     detected_items, visualization = abnormality_detection.judge_image(
         model, image, allowed_regions, device)
```

Comment:
You forgot to pass conf_threshold as a parameter of `judge_image`
```suggestion
    detected_items, visualization = abnormality_detection.judge_image(
        model, image, allowed_regions, device, conf_threshold=args.conf_threshold)
```

You also need to update `abnormality_detection.judge_image` by adding conf_threshold as argument and passing forward to grasping_infer:

```
def judge_image(model: Dict[str, Any], image: Image.Image, allowed_regions: np.ndarray, device: torch.device,
                conf_threshold: float = 0) -> List[Dict[str, Any]]:

    # call grasping inference
    detected_objects, visualization = grasping_inference(model, image, device, conf_threshold=conf_threshold)
...
```